### PR TITLE
[Android] Bump target SDK to 28

### LINKED ISSUE #4501: [Android] Bump target sdk to 28
# Platform

* Android

# Details

Bump target SDK version to at least 28. Also bump support libraries accordingly.

### PR DESCRIPTION
**Pull request by golddove**
## Related Issue
Fixes #4501 
Will stop dependabot from complaining :) PR's #4070, #4072, #4074 

## Description
Bump Android SDK to 28

Caused two regressions, so those fixes are included in this PR.
- ActionSet's failed to show their children
- Images failed to load due to Android security permission changes

## How Verified
Manually compared all `Element` and `Scenario` cards in `v1.0` to `v1.3` for regressions

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/4605)

### CODE REVIEW

File: source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/action/ActionSetRenderer.java [hunk 1]
```diff
@@ -54,7 +54,7 @@ public View render(
         LinearLayout rootLayout = new LinearLayout(context);
         // Add tag for rendering process
         rootLayout.setTag(new TagContent(actionSet));
-        rootLayout.setLayoutParams(new LinearLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.MATCH_PARENT));
+        rootLayout.setLayoutParams(new LinearLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT));
```
Comment (almedina-ms) [inline source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/action/ActionSetRenderer.java:57]: The changes in this file are not related to this issue

Comment (golddove) [reply source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/action/ActionSetRenderer.java:57]: @almedina-ms 

As per my PR description above - it was to address a regression that occurred when I bump the SDK/support library to 28. ActionSet rendering broke. Android SDK may have made changes to its layout behavior, I'm not exactly sure.
